server.py

```diff
@@ -8,7 +9,35 @@ def desk():
     return render_template("index.html")
 
 
-# def ask(word):#
+@app.route("/tagged/word/%string word%")
+def req():

```

Comment:
Don't you think, methods with wildcards must take some parameter? It will be better than to use `requests.args.get` function.
```suggestion
def req(word=None):
```